browser/css/notebookbar.css

```diff
@@ -202,9 +202,6 @@
 /* unobuttons */
 
 .hasnotebookbar .ui-content .unobutton {
-	width: 32px;
-	height: 32px;
-	margin-inline-end: 5px;
```

Comment:
Better to add this to another commit. It is unrelated to this commit. Let's please add another with this in and why it is not needed (outdated rules,etc, now being overruled by more recent changes in notebookbar.css:77 etc)